render/template.go

```diff
@@ -67,6 +74,60 @@ func (s templateRenderer) exec(name string, data Data) (template.HTML, error) {
 	return template.HTML(body), nil
 }
 
+func (s templateRenderer) addAssetsHelpers(helpers map[string]interface{}) map[string]interface{} {
+	helpers["assetPath"] = func(file string) template.HTML {
+		return template.HTML(s.assetPath(file))
+	}
+
+	helpers["javascriptTag"] = func(file string) template.HTML {
+		jsTag := tags.New("script", tags.Options{
+			"type": "text/javascript",
+			"src":  s.assetPath(file),
+		})
+
+		return jsTag.HTML()
+	}
+
+	helpers["stylesheetTag"] = func(file string) template.HTML {
+		cssTag := tags.New("link", tags.Options{
+			"rel":   "stylesheet",
+			"media": "screen",
+			"href":  s.assetPath(file),
+		})
+
+		return cssTag.HTML()
+	}
+
+	return helpers
+}
+
+func (s templateRenderer) assetPath(file string) string {
+	manifest, err := s.AssetsBox.MustString("manifest.json")
+	if err != nil {
+		log.Println("[INFO] didn't find manifest, using raw path to assets")
+
+		return fmt.Sprintf("assets/%v", file)
+	}
+
+	var manifestData map[string]string
+	err = json.Unmarshal([]byte(manifest), &manifestData)
+
+	if err != nil {
+		log.Println("[Warning] seems your manifest is not correct")
+		return ""
+	}
+
+	if file == "application.css" {
```

Comment:
The key the manifest generates is called `main.js` instead of `application.js` i've not found much info about how to change this with webpack on the web